spec/parts/linux/cloud-init/artifacts/cse_install_spec.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/bash
+
+source parts/linux/cloud-init/artifacts/cse_helpers.sh
+
+readPackage() {
+    local packageName=$1
+    packages=$(jq ".Packages" "./spec/parts/linux/cloud-init/artifacts/test_components.json" | jq ".[] | select(.name == \"$packageName\")")
+    echo "$packages"
+}
+
+Describe 'cse_install.sh'
```

Comment:
assuming these are unit tests of some sort, can we add a readme to explain what this is/how they work?

Reply:
ah nvm , I see you updated the readme at the root